linter/allowlists/en.txt

```diff
@@ -11,6 +11,7 @@ assets
 Assistant
 Avatar
 automator
+backend
```

Comment:
That's incorrect. The correct English term is `back end` ;)

Reply:
I adapted it in my description, I did not remove it from the whitelist, because other extensions use it as a keyword.

Reply:
Which would be incorrect imho but whatever.